cluster/gce/util.sh

```diff
@@ -585,11 +585,12 @@ function kube-up {
     --zone "${ZONE}" \
     --size "10GB"
 
-  # Generate a bearer token for this cluster. We push this separately
-  # from the other cluster variables so that the client (this
+  # Generate a bearer token for kubelets in this cluster. We push this
+  # separately from the other cluster variables so that the client (this
   # computer) can forget it later. This should disappear with
   # https://github.com/GoogleCloudPlatform/kubernetes/issues/3168
   KUBELET_TOKEN=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  KUBE_PROXY_TOKEN=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
```

Comment:
Oh. Except I'm not paying attention and you're generating a second token. I need more coffee.

GKE currently bakes these into the kube-env, as I just noted. So this would break us, and we'd wait forever (if I'm reading the code correctly).